runtime/oti/vm_api.h

```diff
@@ -4380,10 +4380,11 @@ fatalRecursiveStackOverflow(J9VMThread *currentThread);
 * @param privateFlags
 * @param memorySpace
 * @param threadObject the object associated with this thread, or NULL
+* @param threadName string representing thread name, or NULL
```

Comment:
What is this (`allocateVMThread` has no such parameter)?

Reply:
Removed the parameter in question.

cherry-picked 
* https://github.com/eclipse-openj9/openj9/pull/19426
Also updated `toggleSuspendOnJavaThreads()` usage & function comments.